src/jsonv-tests/serialization_builder_tests.cpp

```diff
@@ -29,12 +29,17 @@ namespace
 struct person
 {
     person() = default;
-    
+
     person(std::string       f,
            std::string       l,
            int               a,
+#ifdef __APPLE__
```

Comment:
Let's just use the `#ifdef __APPLE__` part of this -- the `{}` are just for convenience, so it doesn't really matter in a unit test.

Strange that the `{}`s don't work...I thought OSX was LLVM 3.5.